articles/inter-vlan-routing/index.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: engineering-education
+
+
+
+status: publish
+published: true
+url: /engineering-education/inter-vlan-routing/
+title: Inter-VLAN routing
+description: This article is viewed at helping one to gain the understanding of inter-vlan routing and how to configure it using a packet tracer, add fields `dateCreated` and `lastModified` in the `Todo` data class.
+author: atonya-dennis
+date: 2021-03-16T00:00:00-10:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/inter-vlan-routing/hero.jpg
+    alt: Network Connection example image
+---
+
+As a network administrator, one of the tasks is to segment a single switched Local Area Network (LAN) logically without having to run new cables and changing the existing network. The concept of "VLAN" and its configurations will help one handle the task at hand with ease.
+<!--more-->
+In this article, we will learn about what inter-VLAN routing is, various methods for VLAN routing, and also build a network configuration to understand how it works.
+
+# Table of contents
+1. [Prerequisites.](#prerequisites)
+2. [Vlan routing terminology.](#terminology)
+3. [Inter VLAN routing.](#intervlanrouting)
+    1. [Methods of Inter VLAN  routing.](#methods)
+4. [Inter VLAN Network Configurations.](#configurations)
+5. [Conclusion. ](#conclusion)
+
+### Prerequisites <a name="prerequisites"></a>
```

Comment:
Don't add these anchor tags `<a>`. These are not needed when you use as I mentioned in my previous comment.

I believe you're using some online text editor to write these codes. I suggest you to use notion.so, which does the markdown conversion easily.

Reply:
Remove them wherever needed.